nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GenerateTableFetch.java

```diff
@@ -112,6 +112,17 @@
             .addValidator(StandardValidators.NON_NEGATIVE_INTEGER_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor RIGHT_BOUND_WHERE = new PropertyDescriptor.Builder()
```

Comment:
I am OK with proceeding that way, though I'd feel better if I knew how many databases this has been tested on. When I wrote it my focus was on one, relatively uncommon (SAP HANA) system. It tests out fine, but I just worry about making it the default.